src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java

```diff
@@ -145,6 +143,18 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                         group.getPermissions().clear();
                         MessageUtils.sendSuccessMessage("Cleared all permissions from the group: " + group.getName(), channel);
                         return;
+                    } else if (args[2].equalsIgnoreCase("move")) {
+                        if(args.length >= 4) {
```

Comment:
Don't nest this if or use an else statement. Your call. After that this is good to merge.

Reply:
@WalshyDev we have it like that literally everywhere else........ It's even in our style guild to do it this way

Reply:
No... we don't anywhere say or use nested if statements

Reply:
Then why have i gotten yelled at before for using not using nested ifs. I mean look at the entire permissions command

Reply:
... we have never said to use nested ifs, if there's no point it being nested don't make it nested. Fix the perm command if you wish then

Reply:
Fine whatever you say, but I have been told to use nested ifs before.